### PR TITLE
preprocess_example_module: Drupal 9 compat

### PR DESCRIPTION
**Pull request by Ambient-Impact**
So that Upgrade Status stops bothering me. :P

### CODE REVIEW

File: examples/preprocess_example_module/preprocess_example_module.info.yml [hunk 1]
```diff
@@ -2,4 +2,5 @@ name: Preprocess example module
 type: module
 description: Registers an extra preprocess event.
 core: 8.x
```
Comment (pdenooijer) [inline examples/preprocess_example_module/preprocess_example_module.info.yml:4]: This one can go right?

Comment (Ambient-Impact) [reply examples/preprocess_example_module/preprocess_example_module.info.yml:4]: Probably, though I would recommend making sure that the minimum core version for the whole project [is 8.7.7 for when they added ```core_version_requirement```](https://www.drupal.org/node/3070687), just so you don't end up with the one person who still hasn't updated Drupal opening an issue about it failing. :P

Edit: oh right, it looks like if you just remove ```core: 8.x```, that'll have the intended effect so forget what I said. Carry on.

Comment (pdenooijer) [reply examples/preprocess_example_module/preprocess_example_module.info.yml:4]: I removed all the other `core` lines from the info files already 😉. This one is not a real module so basically forgot 😅.

Comment (Ambient-Impact) [reply examples/preprocess_example_module/preprocess_example_module.info.yml:4]: Fair enough!

**Comment by pdenooijer** [comment]
Version 2.03 has the above fix and some other fixed 😁

**Pull request merged**